baaas-decision-fleet-manager/src/main/java/org/kie/baaas/dfm/app/config/DecisionFleetManagerConfig.java

```diff
@@ -48,6 +48,10 @@
     @ConfigProperty(name = "baaas.dfm.s3.bucket.name")
     String bucketName;
 
+    @Inject
+    @ConfigProperty(name = "baaas.dfm.max.allowed.decisions")
+    int maxAllowedDecisions = -1; //disabled by default
```

Comment:
Is it easier to make this a `long`? I see you have to do some casting between `int` and `long` in the validator?